### PR TITLE
Minor code cleanup and documentation fixes

### PR DESCRIPTION
**Pull request by Nyholm**
Removed use of container parameters and properly register the command as a service. I also helped the auto write feature to find our services.

### CODE REVIEW

File: DependencyInjection/EkinoNewRelicExtension.php [hunk 1]
```diff
@@ -39,22 +39,23 @@ public function load(array $configs, ContainerBuilder $container)
...
-        if ($config['logging'])
-        {
+        if ($config['logging']) {
             $container->setAlias('ekino.new_relic.interactor', 'ekino.new_relic.interactor.logger');
             $container->getDefinition('ekino.new_relic.interactor.logger')
                 ->replaceArgument(0, new Reference($interactor));
         }
-        else
-        {
+        else {
```
Comment (jderusse) [inline DependencyInjection/EkinoNewRelicExtension.php]: shouldn't it be `} else {`?

File: Resources/config/services.xml [hunk 2]
```diff
@@ -4,13 +4,6 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
-    <parameters>
-        <parameter key="ekino.new_relic.interactor.real.class">Ekino\Bundle\NewRelicBundle\NewRelic\NewRelicInteractor</parameter>
```
Comment (jderusse) [inline Resources/config/services.xml]: :+1:

File: DependencyInjection/EkinoNewRelicExtension.php (hunk 1 above)
Comment (Nyholm) [reply DependencyInjection/EkinoNewRelicExtension.php]: Yeah, True. I was not supposed to do CS fixes but that one really hurt my eyes. =)

Thanks.

**Comment by Nyholm** [comment]
Thank you for the review

File: DependencyInjection/EkinoNewRelicExtension.php (hunk 1 above)
Comment (jderusse) [reply DependencyInjection/EkinoNewRelicExtension.php]:
```
}
else {
```

hurts much more my eyes ;-)

What's about adding a php-cs-fixer with basic rules?

Comment (Nyholm) [reply DependencyInjection/EkinoNewRelicExtension.php]: =)

Excellent idea, I've created a PR: https://github.com/ekino/EkinoNewRelicBundle/issues/142

**Pull request merged**